client/rest-high-level/src/test/java/org/elasticsearch/client/MachineLearningIT.java

```diff
@@ -2143,15 +2142,19 @@ private static XContentBuilder mappingForClassification() throws IOException {
         .endObject();
     }
 
-    private static IndexRequest docForClassification(String indexName, String actualClass, String predictedClass, double p) {
+    private static IndexRequest docForClassification(String indexName,
+                                                     String actualClass,
+                                                     String predictedClass,
+                                                     double p,
+                                                     String otherClass) {
         return new IndexRequest()
             .index(indexName)
             .source(XContentType.JSON,
                 actualClassField, actualClass,
                 predictedClassField, predictedClass,
                 topClassesField, List.of(
                     Map.of("class_name", predictedClass, "class_probability", p),
-                    Map.of("class_name", "other", "class_probability", 1 - p)));
+                    Map.of("class_name", otherClass, "class_probability", 1 - p)));
```

Comment:
"well actually" 1-p doesn't seem correct here as there could be more than two classes. 

But, from what I tell, this doesn't effect the testing, right?